modules/govuk_elasticsearch/manifests/plugins.pp

```diff
@@ -15,16 +15,17 @@
     instances  => $::fqdn,
   }
 
-  case $govuk_elasticsearch::version {
-    /^1.4/:  { $cloud_aws_version = '2.4.2' }
-    /^1.5/:  { $cloud_aws_version = '2.5.1' }
-    /^1.7/:  { $cloud_aws_version = '2.7.1' }
-    default: { fail('Not able to select a version for cloud-aws, see https://github.com/elastic/elasticsearch-cloud-aws') }
-  }
+  if $govuk_elasticsearch::version =~ /^1\./ {
+    case $govuk_elasticsearch::version {
+      /^1.4/:  { $cloud_aws_version = '2.4.2' }
+      /^1.5/:  { $cloud_aws_version = '2.5.1' }
+      /^1.7/:  { $cloud_aws_version = '2.7.1' }
+      default: { fail('Not able to select a version for cloud-aws, see https://github.com/elastic/elasticsearch-cloud-aws') }
+    }
 
-  elasticsearch::plugin { "elasticsearch/elasticsearch-cloud-aws/${cloud_aws_version}":
-    module_dir => 'cloud-aws',
-    instances  => $::fqdn,
+    elasticsearch::plugin { "elasticsearch/elasticsearch-cloud-aws/${cloud_aws_version}":
```

Comment:
instead of doing this can we load a version for ES 2.4 

looking at: 

https://www.elastic.co/guide/en/elasticsearch/plugins/2.4/cloud-aws.html

I think it might be :

```
elasticsearch::plugin { "cloud-aws":
  module_dir => 'cloud-aws',
  instances  => $::fqdn,
}
```